### ISSUE TITLE
Sharing plain text

### ISSUE THREAD

**Issue by durdevic** [BigPun86/React-Native-ShareExtensionExample#2]
Hey man, thanks for making this example, works nicely on iOS. 

Quick question, how would you transfer the text to the application without actually going in the app. I'm trying to share a URL from the browser to the my application, and I just want a small popup window to overlay the browser and when I click close that he saves it in my app. 

I thought I might use Redux, but the provider for the app is separated. 

In short how can I transfer data between second and main application? 

Thanks 👋
